src/renderer/lib/hooks/useAppListeners.ts

```diff
@@ -193,4 +198,14 @@ export const useAppListeners = () => {
   React.useEffect(() => {
     init(rootSlpPath, extraSlpPaths, true).catch(console.error);
   }, [rootSlpPath, extraSlpPaths, init]);
+
+  const [startBroadcast] = useBroadcast();
+  ipc_broadcastReconnect.renderer!.useEvent(async ({ config }) => {
+    startBroadcast(config).catch(log.error);
+  }, []);
+
+  const [, refreshBroadcasts] = useBroadcastList();
+  ipc_spectateReconnect.renderer!.useEvent(async () => {
+    refreshBroadcasts()?.catch(log.error);
```

Comment:
Why do you have an optional chain here?

```suggestion
    refreshBroadcasts().catch(log.error);
```

Reply:
it is a `Promise<void> | undefined` type, I could do a quick check.